src/pages/patch/ParametersModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { Modal } from "components/Modal";
+import { Parameter } from "types/patch";
+
+interface ParametersProps {
+  parameters: Parameter[];
+  dataCy: string;
+  closeModal: () => void;
+  visible: boolean;
+}
+export const ParametersModal: React.FC<ParametersProps> = ({
+  parameters,
+  dataCy,
+  closeModal,
+  visible,
+}) => (
+  <Modal
+    visible={visible}
+    footer={null}
+    onCancel={closeModal}
+    data-cy={dataCy}
+    title="Patch Parameters"
+  >
+    <span>
```

Comment:
is this span necessary?

Reply:
maybe just a failure of copy/paste haha.